sbor/src/instruction.rs

```diff
@@ -1,4 +1,4 @@
-// Instructions recognized by instruction extractor
+/// Instructions recognized by instruction extractor
```

Comment:
incorrect doc, there is an empty space between this doc and the type `enum Instruction`, the doc must go on top of `#[repr()u8]`, without newline between. And **every line in the doc must be `///`**